src/docs/product/integrations/cloud-monitoring/aws-lambda/index.mdx

```diff
@@ -154,18 +156,20 @@ functions:
   <yourFunctionName>:
     handler: sentry_sdk.integrations.init_serverless_sdk.sentry_lambda_handler
     environment:
-      SENTRY_INITIAL_HANDLER: '<yourLambdaHandlerFunctionPath>'
+      SENTRY_INITIAL_HANDLER: "<yourLambdaHandlerFunctionPath>"
     layers: ${self:custom.layers}
 ```
 
 Currently there is no way of using the [latest layer plugin](https://www.npmjs.com/package/serverless-latest-layer-version) since the lambda:ListLayerVersions on resource: arn:aws:lambda:eu-west-1:943013980633:layer:SentryNodeServerlessSDK has not been given the necessary permissions.
 It is a relatively new integration but there is definitely a case to write a serverless plugin to make this super-smooth. There is an [older version](https://www.serverless.com/plugins/serverless-sentry-plugin) but it requires code changes to the handler and doesn't use `NODE_OPTIONS`. A new plugin could also add the Sentry layer to every function automatically.
```

Comment:
```suggestion
Currently, there's no way of using the [latest layer plugin](https://www.npmjs.com/package/serverless-latest-layer-version) since the `lambda:ListLayerVersions on resource: arn:aws:lambda:eu-west-1:943013980633:layer:SentryNodeServerlessSDK` hasn't been given the necessary permissions.

```
I don't think it makes sense to add this here since it's not really actionable for users and sounds more like internal notes, unless I'm misunderstanding something.